themes/10up-theme/assets/css/frontend/02-global/reset.css

```
@@ -0,0 +1,83 @@
+/**
+* Reset.css
+* Custom reset enhancments.
+* Primary reset is modern-normalize (by default)
+* imported in ../style.css
+*/
+
+/*
+ * Reset spacing & font inheritance
+ *
+ * Font iheriance: We add intentional font inheritence as there are browser
+ * inconstencies as well as some instances where lacking implecit inheritence
+ * can cause challenges
+ *
+ * Margin & Padding reset: We reset margin and padding to 0 to ensure a
+ * consistentstarting point for all elements
+ */
+* {
+	font: inherit;
+	margin: 0;
+	padding: 0;
+}
+
+html {
+
+	/* enable dark and light color schemes by default */
+	color-scheme: dark light; /* stylelint-disable-line scale-unlimited/declaration-strict-value */
+
+	/* enable hanging punctuation */
+	hanging-punctuation: first last; /* stylelint-disable-line scale-unlimited/declaration-strict-value */
+}
+
+/*
+ * Ensure body is at least 100% of viewport height
+ */
+body {
+	min-height: 100svh;
```

Shouldn't we be using `dvh` here? (And provide a fallback for when the new unit isn't available?)

```suggestion
	min-height: 100%;
	min-height: 100dvh;
```